src/engine/source/base/parseEvent.cpp

```diff
@@ -107,7 +107,7 @@ Event parseOssecEvent(const std::string& event)
             if (registeredIP.find(':') != std::string::npos)
             {
                 // IPv6 case
-                endIdx = event.find(":", endIdx + 2);
+                endIdx = event.find(':', endIdx + 2);
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#performance

'find' called with a string literal consisting of a single character; use the more effective overload accepting a character